### PR TITLE
[ja]Add note to ResourceQuota docs

### PR DESCRIPTION
**Pull request by YuikoTakada**
This PR adds note to [RessourceQuota docs](https://kubernetes.io/ja/docs/concepts/policy/resource-quotas/).
This change is following to [this commit](https://github.com/kubernetes/website/commit/6abdc256ad33cd0d3438addfc0b9f03ab7754c14).
FYI: https://kubernetes.io/docs/concepts/policy/resource-quotas/

### CODE REVIEW

File: content/ja/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ weight: 10
 - リソースの作成や更新がクォータの制約に違反しているとき、そのリクエストはHTTPステータスコード`403 FORBIDDEN`で失敗し、違反した制約を説明するメッセージが表示されます。
 - `cpu`や`memory`といったコンピューターリソースに対するクォータが名前空間内で有効になっているとき、ユーザーはそれらの値に対する`requests`や`limits`を設定する必要があります。設定しないとクォータシステムがPodの作成を拒否します。 ヒント: コンピュートリソースの要求を設定しないPodに対してデフォルト値を強制するために、`LimitRanger`アドミッションコントローラーを使用してください。この問題を解決する例は[walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)で参照できます。
 
+{{< note >}}
+- `cpu`および`memory`リソースに関しては、ResourceQuotaはlimitを設定しているnamespace内の**全ての**（新しい）Podに対して当該リソースのlimitの設定を強制します。namespace内の`cpu`または`memory`どちらかに対してリソースクォータを適用する場合、ユーザやクライアントは全ての新しいPodごとに、そのリソースの`requests`あるいは`limits`を指定**しなければなりません** そうしないと、コントロールプレーンがそのPodの作成を拒否する可能性があります。
+- その他のリソースに関して: ResourceQuotaはリソースのlimitまたはrequestを設定していないpodの場合でも機能します。これは、リソースクォータがnamespace内のエフェメラルストレージに対してlimitを設定している場合、エフェメラルストレージのlimit/requestsを設定していないpodを新規作成できることを意味します。[LimitRange](/docs/concepts/policy/limit-range/)を使うことで、リソースに対して自動的にデフォルト要求を設定することができます。
```
Comment (atoato88) [inline content/ja/docs/concepts/policy/resource-quotas.md:30]:
```suggestion
- その他のリソースに関して: ResourceQuotaはリソースのlimitまたはrequestを設定していないPodの場合でも機能します。これは、リソースクォータがnamespace内のエフェメラルストレージに対してlimitを設定している場合、エフェメラルストレージのlimit/requestsを設定していないPodを新規作成できることを意味します。[LimitRange](/ja/docs/concepts/policy/limit-range/)を使うことで、リソースに対して自動的にデフォルト要求を設定することができます。
```
- We have the `ja` content of this page, so link to that.
- We should use `Pod` not `pod` I think.

**Review by atoato88 — COMMENTED** [review]
Please check comment.

**Comment by atoato88** [comment]
And another comment:

We should use `名前空間` not `namespace`?
Because that other part of this page use `名前空間` about `namespace` in en content.

File: content/ja/docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -25,6 +25,11 @@ weight: 10
 - リソースの作成や更新がクォータの制約に違反しているとき、そのリクエストはHTTPステータスコード`403 FORBIDDEN`で失敗し、違反した制約を説明するメッセージが表示されます。
 - `cpu`や`memory`といったコンピューターリソースに対するクォータが名前空間内で有効になっているとき、ユーザーはそれらの値に対する`requests`や`limits`を設定する必要があります。設定しないとクォータシステムがPodの作成を拒否します。 ヒント: コンピュートリソースの要求を設定しないPodに対してデフォルト値を強制するために、`LimitRanger`アドミッションコントローラーを使用してください。この問題を解決する例は[walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)で参照できます。
 
+{{< note >}}
+- `cpu`および`memory`リソースに関しては、ResourceQuotaはlimitを設定しているnamespace内の**全ての**（新しい）Podに対して当該リソースのlimitの設定を強制します。namespace内の`cpu`または`memory`どちらかに対してリソースクォータを適用する場合、ユーザやクライアントは全ての新しいPodごとに、そのリソースの`requests`あるいは`limits`を指定**しなければなりません** そうしないと、コントロールプレーンがそのPodの作成を拒否する可能性があります。
```
Comment (t-inu) [inline content/ja/docs/concepts/policy/resource-quotas.md:29]:
```suggestion
- `cpu`および`memory`リソースに関しては、ResourceQuotaはlimitを設定しているnamespace内の**全ての**(新しい)Podに対して当該リソースのlimitの設定を強制します。namespace内の`cpu`または`memory`どちらかに対してリソースクォータを適用する場合、ユーザーやクライアントは全ての新しいPodごとに、そのリソースの`requests`あるいは`limits`を指定**しなければなりません**。そうしないと、コントロールプレーンがそのPodの作成を拒否する可能性があります。
```

File: content/ja/docs/concepts/policy/resource-quotas.md (hunk 1 above)
Comment (YuikoTakada) [reply content/ja/docs/concepts/policy/resource-quotas.md:30]: done

**Comment by YuikoTakada** [comment]
> 名前空間

done

**Comment by YuikoTakada** [comment]
@atoato88 @t-inu Thank you for your comments. I've fixed them, PTAL

**Comment by t-inu** [comment]
/assign @nasa9084
Please review this PR.

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by nasa9084** [comment]
sorry being late~
/approve

**Pull request merged**